UPGRADE-4.3.md

```diff
@@ -123,6 +123,7 @@ Routing
    options have been deprecated.
  * Implementing `Serializable` for `Route` and `CompiledRoute` is deprecated; if you serialize them, please
    ensure your unserialization logic can recover from a failure related to an updated serialization format
+ * Not tagging the route loader services with `routing.router_loader` has been deprecated.
```

Comment:
`route loader` vs `router_loader`

Reply:
I think router loader is way better.

Reply:
In this case, if the "public" name is now router loader, I guess we should update the whole documentation page (https://symfony.com/doc/current/routing/custom_route_loader.html)

Reply:
and I meant `route loader`, sorry

Reply:
Do you mean you want to move back to the original tag name `routing.route_loader`?

Reply:
Works for me, let's rename the interface + existing implementation, I suggest deprecating `ServiceRouterLoader` in favor of `ContainerRouteLoader`.

Reply:
WDYT of renaming everything in another PR after this one is merged for clarity ? First, finish this one (I just need to rename the tag back to `routing.route_loader`). Then, another one of pure refacto.

Reply:
friendly ping @fabpot @chalasr, would be cool to have this in 4.3